compendium/DeclarativeServices/src/manager/BundleLoader.cpp

```diff
@@ -110,6 +110,10 @@ GetComponentCreatorDeletors(
                     ToString(fromBundle.GetBundleId()) +
                     " (location=" + bundleLoc + ")");
     } catch (const std::system_error& ex) {
+      logger->Log(logservice::SeverityLevel::LOG_INFO,
```

Comment:
I think we want to use the `Log` overload that takes an `exception_ptr` so that the exception is also logged.